daprdocs/content/en/operations/monitoring/jaeger.md

```diff
@@ -0,0 +1,106 @@
+---
+type: docs
+title: "How-To: Set up Jaeger for distributed tracing"
+linkTitle: "Jaeger"
+weight: 3000
+description: "Set up Jaeger for distributed tracing"
+type: docs
+---
+
+The following steps shows you how to configure Dapr to send distributed tracing data to Jaeger running as a container in your Kubernetes cluster, how to view them.
+
+## Setup
+
+First the following YAML file to install Jaeger
+* jaeger-operator.yaml
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "jaeger"
+spec:
+  strategy: allInOne
+  ingress:
+    enabled: false
+  allInOne:
+    image: jaegertracing/all-in-one:1.13
+    options:
+      query:
+        base-path: /jaeger
+```
+
+Now, let's use that Jaeger Operator resource to install Jaeger
```

Comment:
please remove use of first person. For example change to "Now, use the above YAML file to install Jaeger"